Modules/Numerics/FEM/include/itkFEMLinearSystemWrapperDenseVNL.h

```diff
@@ -151,24 +151,24 @@ class ITKFEM_EXPORT LinearSystemWrapperDenseVNL : public LinearSystemWrapper
   ScaleSolution(Float scale, unsigned int solutionIndex);
 
   void
-  SwapMatrices(unsigned int matrixIndex1, unsigned int matrixIndex2) override;
+  SwapMatrices(unsigned int MatrixIndex1, unsigned int MatrixIndex2) override;
```

Comment:
The convention for parameter names it to begin with lowercase letter, if I remember correctly. Name synchronization in FEM module should be done the other way.